### PR TITLE
session: tiny update leak check for some test function

### PR DESCRIPTION
**Pull request by tiancaiamao**
1. Don't skip TestSysSessionPoolGoroutineLeak function
2. Fix TestSchemaCheckerSimple testleak

@XuHuaiyu @coocood

### CODE REVIEW

File: session/tidb_test.go [hunk 1]
```diff
@@ -164,21 +160,9 @@ func (s *testMainSuite) TestSysSessionPoolGoroutineLeak(c *C) {
...
-			return
-		}
-		time.Sleep(20 * time.Millisecond)
-	}
-	after := runtime.NumGoroutine()
-	c.Assert(after-before, Less, 3)
 }
 
 func (s *testMainSuite) TestSchemaCheckerSimple(c *C) {
-	defer testleak.AfterTest(c)()
```
Comment (shenli) [inline session/tidb_test.go]: Why remove this?

Comment (zimulala) [reply session/tidb_test.go]: Done it in `TearDownSuite`

**Comment by coocood** [comment]
LGTM

**Pull request merged**